cpp/models/abm/README.md

```diff
@@ -1,57 +1,127 @@
 
-First each person interacts with the other persons at the same location. This interaction determines their infection state transitions. A susceptible person may become exposed by contact with an infected person without symptoms. The probability of infection depends on a multitude of factors, such as the viral load and infectiousness of the infectee and the immunity level of the susceptible person.
+Another way of migration we use in the simulation of Braunschweig (simulations/abm_braunschweig.cpp) is using trips. A trip consists of the ID of the person that performs this trip, a timepoint when this trip is performed and where the person is heading to. In the beginning of the simulation a list with all trips is initialized and followed during the simulation. There can be different trips on the weekend than during the week but other than that the agents do the same trips every day. As before agents that are in quarantine or in the hospital cannot migrate.
 
-During the migration phase, each person may change location. Migration follows complex rules, taking into account the current location, time of day, and properties of the person (e.g. age). Some location changes are deterministic and regular (e.g. going to work), others are random (e.g. going to shopping or to a social event in the evening/on the weekend).
+## Get Started
 
-The result of the simulation is for each time step the count of persons in each infection state at that time.
+This section gives an introduction how to use the ABM and setup your own simulation. For a quick overview can find a full example in the [abm minimal example](../../examples/abm_minimal.cpp) and a more detailed doxygen documentation [here](https://scicompmod.github.io/memilio/documentation/index.html ). For a guide on installation and running the simulations and examples see this [README](../../README.md).
 
-## Example
-An example can also be found at simulations/abm.cpp
+Every person in the ABM belongs to an AgeGroup which we can define:
+
+```cpp
+size_t NUM_AGE_GROUPS         = 4;
+const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(0);
 ```
-    size_t NUM_AGE_GROUPS         = 4;
-    const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(NUM_AGE_GROUPS - 4);
-    const auto AGE_GROUP_5_TO_14  = mio::AgeGroup(NUM_AGE_GROUPS - 3);
-    const auto AGE_GROUP_15_TO_34 = mio::AgeGroup(NUM_AGE_GROUPS - 2);
-    const auto AGE_GROUP_35_TO_59 = mio::AgeGroup(NUM_AGE_GROUPS - 1);
 
-    // Create the world with 4 age groups.
-    auto world = mio::abm::World(NUM_AGE_GROUPS);
-    //setup the parameters
-    world.parameters.get<mio::abm::IncubationPeriod>() = 4.;
-    //...
+The initial empty world is created with the number of age groups:
+
+```cpp
+auto world = mio::abm::World(NUM_AGE_GROUPS);
```

Comment:
Please mention that all age groups must have values striclty smaller than num_age_groups, i.e. values in [0, num_age_groups).

Reply:
did that